tests/baselines/reference/bestChoiceType.types

```diff
@@ -4,7 +4,7 @@
 // Repro from #10041
 
 (''.match(/ /) || []).map(s => s.toLowerCase());
->(''.match(/ /) || []).map(s => s.toLowerCase()) : any[]
+>(''.match(/ /) || []).map(s => s.toLowerCase()) : string[]

```

Comment:
The change from `any` to `string` is an unexpected bonus.  Probably the `any` was due to an inviable  `getSignaturesOfStructuredType` result.